optimum/exporters/tasks.py

```diff
@@ -18,12 +18,14 @@
 import inspect
 import itertools
 import os
+import warnings
 from functools import partial
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Set, Tuple, Type, Union
 
 import huggingface_hub
 from huggingface_hub.constants import HUGGINGFACE_HUB_CACHE
+from huggingface_hub.utils import OfflineModeIsEnabled
```

Comment:
Good catch, thank you for fixing. Maybe even move to `huggingface_hub.errors.OfflineModeIsEnabled`? https://github.com/huggingface/huggingface_hub/blob/v0.23.3/src/huggingface_hub/errors.py#L16

As I guess the `huggingface_hub.utils.OfflineModeIsEnabled` path is deprecated?